### ISSUE TITLE
Gimp: fonts not available

### ISSUE THREAD

**Issue by tristelune1** [kisslinux/community#123]
Hello, 

I have created a package for `gimp`, it starts, I can use it, but when I try to add text to a picture, I get the following:

     Due to lack of any fonts, text functionality is not available.

On my system I have: `dejavu`, `liberation`. As I have seen, I need to run `fc-cache` such that the fonts will be found. The must be available in `/var/cache/fontconfig`. There, I have the following:

````
ls -l /var/cache/fontconfig/
total 80
-rw-r--r--    1 root     root         48208 Nov 22 23:35 2c204173d3592afc30a350ed81f64c67-le64.cache-7
-rw-r--r--    1 root     root           168 Nov 22 23:35 3830d5c3ddfd5cd38a049b759396e72e-le64.cache-7
-rw-r--r--    1 root     root           200 Dec 18 20:34 CACHEDIR.TAG
-rw-r--r--    1 root     root           144 Nov 22 23:35 c855463f699352c367813e37f3f70ea7-le64.cache-7
-rw-r--r--    1 root     root         20064 Nov 22 23:35 f6b893a7224233d96cb72fd88691c0b4-le64.cache-7
````

I have run `fc-cache`, it didn't help. 

My build file:

````
#!/bin/sh -e

patch -p1 < fix_intltool.patch
patch -p1 < remove_glib_networking.patch
patch -p1 < remove_po_dir.patch

sed -i -e '/$(helpbrowser)/d' plug-ins/Makefile.in

./configure \
    --prefix=/usr \
    --libexecdir=/usr/bin \
    --disable-altivec \
    --disable-static \
    --disable-python \
    --disable-nls \
    --without-lcms \
    --without-webkit

make
make DESTDIR="$1" install
````

Do you need something else ? 

I think there is another problem, but it's possible to use gimp. I can package it, then you could install it and test it. Just tell me what is the best for you. 

Thank you for your help!

**Comment by tristelune1** [comment]
I got no clues on a forum, I have tried on another one with more details:

[Thread in gimp-forum.net](https://www.gimp-forum.net/Thread-Fonts-not-available-in-text-Tool)

If I can't get any help there, I'll try to write a bug report, although I don't think it's a bug. But unfortunately I see no other solution.

**Comment by tristelune1** [comment]
I have tried to packaged `intltool`, `gettext` and `glib-networking`. Unfortunately it's not better. I'll wait a little bit and probably report a bug. More in the thread of the forum.

**Comment by tristelune1** [comment]
This problem has been reported on Gitlab:

[issue 4478](https://gitlab.gnome.org/GNOME/gimp/issues/4478)

If somebody is more experienced, I will probably need the help. I can provide the local packages I have.

**Comment by tristelune1** [comment]
That was a long research....., I don't know why I didn't look at this:

[Fedora, no fonts available](https://bugzilla.redhat.com/show_bug.cgi?id=1744377)

The patch is here:

[patch for fontconfig](https://src.fedoraproject.org/rpms/fontconfig/blob/master/f/fontconfig-fix-1744377.patch)

I have added the patch locally to `fontconfig` and I can use the text tool in `gimp`. :smiley:. 

@dylanaraps : could you add the patch to `fontconfig` ? 

Then I think it will be possible to finish the packaging.

**Comment by dylanaraps** [comment]
Done. Patch added and pushed. 

Great work tracking down the issue. :+1:
